mypyc/test-data/run-misc.test

```diff
@@ -1108,25 +1108,14 @@ assert not C
 # make the initial import fail
 assert False
 
-class C:
-    def __init__(self):
-        self.x = 1
-        self.y = 2
-def test() -> None:
-    a = C()
 [file driver.py]
 # load native, cause PyInit to be run, create the module but don't finish initializing the globals
-try:
-    import native
-except:
-    pass
-try:
-    # try accessing those globals that were never properly initialized
-    import native
-    native.test()
```

Comment:
I don't think it has done much before. The first `import native` fails, and so the second `import native` also gets executed, and also fails, and we never reached `native.test()`.